lib/ansible/modules/database/postgresql/postgresql_sequence.py

```diff
@@ -0,0 +1,620 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2019, Tobias Birkefeld (@tcraxs) <t@craxs.de>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+DOCUMENTATION = r'''
+---
+module: postgresql_sequence
+short_description: Create, drop, or alter a PostgreSQL sequence
+description:
+- Allows to create, drop or change the definition of a sequence generator
+  U(https://www.postgresql.org/docs/current/sql-createsequence.html).
+version_added: '2.9'
+options:
+  sequence:
+    description:
+    - The name of the sequence.
+    required: true
+    type: str
+    aliases:
+    - name
+  state:
+    description:
+    - The sequence state.
+    - If I(state=absent) other options will be ignored except of I(schema).
```

Comment:
I guess you also need the name?
```suggestion
    - If I(state=absent) other options will be ignored except of I(name) and I(schema).
```

Reply:
@felixfontein thank you for reviewing!
@tcraxs , thank you for your persistence and fixing, if you agree, fix this one, then I add shipit